src/assets/js/search.js

```diff
@@ -98,18 +98,11 @@
   }
 
   const getExcerpt = (content, searchQuery) => {
-    const fullRegex = new RegExp(searchQuery, 'ig')
-    const fullSearchTerm = {
-      term: searchQuery,
-      regex: fullRegex,
-      matchIndex: content.search(fullRegex),
-    }
-
-    const searchTerms = searchQuery.split(' ').filter(i => i).map(term => {
-      const regex = new RegExp(term, 'ig')
-
-      return { term, regex, matchIndex: content.search(regex) }
-    })
+    const fullSearchTerm = getTermMatcher(searchQuery, content)
+    const searchTerms = searchQuery
+      .split(" ")
+      .filter((i) => i)
+      .map((term) => getTermMatcher(term, content));
```

Comment:
Was going to suggest that we keep things consistent in terms of not parenthesising single arguments passed to iterators, following the approach on lines 79 and 117, but I then noticed lines 31 and 72 do the opposite 🤦🏼‍♀️

Seems like Standard doesn't care about this, but maybe in a separate commit it would be good to take a stance either way (I have no strong preference!)

Reply:
Done. I have kept it to removing the parenthesis for single arguments, as I think it is a bit easier to read when on a single line. Line 31 does need a parenthesis though for the array.